ansible_bender/cli.py

```diff
@@ -178,6 +178,14 @@ def _do_build_interface(self):
             "--extra-buildah-from-args",
             help="arguments passed to buildah from command (be careful!)"
         )
+        self.build_parser.add_argument(
+            "--extra-buildah-run-args",
+            help="arguments passed to buildah run command (be careful!)"
+        )
+        self.build_parser.add_argument(
+            "--extra-podman-run-args",
+            help="arguments passed to podman run command (be careful!)"
+        )
```

Comment:
I'm afraid this won't work as:
1. some translation would have to be done: turn a string into list of strings
2. the arguments would need to be escaped somehow so ansible-bender CLI won't interpret them actually

Reply:
please let me know if the CLI interface actually worked for you

Reply:
I haven't actually tried the CLI interface, but I tried to follow the format of how `--extra-ansible-args` worked, going from a single string to a list of strings using [shlex.split()](https://github.com/ansible-community/ansible-bender/blob/cf6b51798ec8b7b32f161e20ca99e1f600378c9d/ansible_bender/core.py#L246). I'm not sure of the specifics around escaping, but these new flags should behave the same way as existing ones. I'll add some tests, try out the cli, and let you know.